src/components/molecules/OnlineStats/OnlineStats.tsx

```diff
@@ -99,27 +99,22 @@ const OnlineStats: React.FC = () => {
   useEffect(() => {
     const liveEvents: Array<VenueEvent> = [];
     const venuesWithAttendance: AttendanceVenueEvent[] = [];
-    const peopleByLastSeen = peopleByLastSeenIn(
-      venueName ?? "",
-      recentVenueUsers
-    );
     openVenues.forEach(
       (venue: {
         venue: WithId<AnyVenue>;
         currentEvents: Array<VenueEvent>;
       }) => {
-        const venueAttendance = peopleAttending(peopleByLastSeen, venue.venue);
         liveEvents.push(...venue.currentEvents);
         venuesWithAttendance.push({
           ...venue,
-          attendance: venueAttendance ? venueAttendance.length : 0,
+          attendance: 0,
```

Comment:
Legacy logic. Can be removed

Reply:
Done in https://github.com/sparkletown/sparkle/pull/2286/commits/1538d6a48a45a2430d78d2624f35a1d12a800aee